Money_Counter_JB.py

```diff
@@ -0,0 +1,44 @@
+# This programme counts British coins based on their weight
```

Comment:
It is actually quite common and a recommended 'style' of python coding to open a file with a [docstring](https://www.python.org/dev/peps/pep-0257/). It works like a comment in the way that it is just for information and not interpreted
```suggestion
"""This programme counts British coins based on their weight."""
```